ci/dockerfiles/Dockerfile.mamba-minimal

```diff
@@ -0,0 +1,21 @@
+FROM quay.io/condaforge/mambaforge
+
+RUN apt update && apt upgrade -y
+RUN DEBIAN_FRONTEND=noninteractive apt install -y tzdata
+
+RUN mamba create -n pandas-dev \
```

Comment:
This is disconnected from requirements-minimal.txt at the moment. Could create a separate env file, break this up into two steps with a pip install, or leave as is